### PR TITLE
fixing schema validation tests

### PR DESCRIPTION
**Pull request by jonnyshaw89**
### What is the context of this PR?
Fixing test_schema_validation and adding a negative test to check if we break the test again

### How to review 
Check that the tests pass

### CODE REVIEW

File: tests/app/data/test_schema_validation.py [hunk 1]
```diff
@@ -13,7 +13,20 @@
...
 class TestSchemaValidation(unittest.TestCase):
 
-    def test_schema(self):
+    def test_invalid_schema(self):
+        schema_file = open(os.path.join(settings.EQ_SCHEMA_DIRECTORY, "schema/schema-v1.json"), encoding="utf8")
+        schema = load(schema_file)
+
+        file = "test_invalid_routing.json"
+
+        json_file = open(os.path.join(settings.EQ_SCHEMA_DIRECTORY + '/../../tests/app/data/', file), encoding="utf8")
```
Comment (benjefferies) [inline tests/app/data/test_schema_validation.py]: Not sure I like the filesystem traversal but not sure what else to suggest

**Review by benjefferies — APPROVED** [review]

File: tests/app/data/test_schema_validation.py (hunk 1 above)
Comment (jonnyshaw89) [reply tests/app/data/test_schema_validation.py]: yeh its the only way i could make it with by running it from any directory e.g. running single test and in run_tests.sh

Comment (jonnyshaw89) [reply tests/app/data/test_schema_validation.py]: However, Travis doesn't like it.

Comment (benjefferies) [reply tests/app/data/test_schema_validation.py]: Other thing I thought about was having the schema in the same directory as the test and find it based on the test_schema_validation.py location

Comment (jonnyshaw89) [reply tests/app/data/test_schema_validation.py]: My bad. was in wrong directory

Comment (jonnyshaw89) [reply tests/app/data/test_schema_validation.py]: how would you do that?

Comment (benjefferies) [reply tests/app/data/test_schema_validation.py]: Oh fair enough. Think you can do something like this `os.path.abspath(__file__)`

**Review by ajmaddaford — APPROVED** [review]

**Pull request merged**